slices/libpam-runtime.yaml

```diff
@@ -20,6 +20,18 @@ slices:
 
   # Applications that need libpam for now should just rely on the config and var slices for now -
   # until appropriate solution for emulating what pam-auth-update can do (or maybe something else).
+  bins:
+    essential:
+      - debconf_bins
+      - libpam-modules_config
+      - libpam-runtime_config
+      - libpam-runtime_pam-config
+      - libpam-runtime_pam-defaults
+      - libpam-runtime_var
+    contents:
+      /usr/sbin/pam-auth-update:
+      /usr/sbin/pam_getenv:
```

Comment:
those comments came from another merged PR. The problem seems to originate from a missing debconf slice, which was added in the current PR. 

As for cron, its is a dependency for logrotate, which is a dependency for rabbitmq-server for non-i386 archs. I see no harm to use the bins here since pam-auth-update should be working with debconf slice included. Correct me if I am missing anything.

Reply:
I ended up removing cron and logrotate since they are only useful for postinst scripts, and we are not doing that in mutation scripts currently.